src/contacts/ViewContact/util.js

```diff
@@ -21,3 +20,13 @@ export const deleteContact = (orgMutator, contactMutator, contactId, org) => {
 
   return Promise.resolve();
 };
+
+export const updateContact = (contactMutator, contact) => {
```

Comment:
what if there is no id, is it still update contact? or maybe save is better?

Reply:
Renamed it